src/app/[locale]/(app)/(nav)/(time-selector)/(top)/top/guilds/details/_components/Stats.tsx

```diff
@@ -16,19 +16,19 @@ export default function Stats({
   const data = [
     {
       value: messagesCount,
-      label: "total messages sent",
+      label: "top.totalMessageSent",
```

Comment:
For all of the labels, call `t` after importing it and adding the "use client" directive